testkit/src/main/resources/reference.conf

```diff
@@ -120,5 +119,48 @@ akka {
 
             event-stream=off
         }
+        default-dispatcher {
+            # The goal here is to reduce the number of threads spun up for test suites.
+            # Since every test suite currently
+            # has it's own actor system, we should limit the number of threads in the system
+            fork-join-executor {
+                # minimum of one thread for tests
+                parallelism-min = 1
+
+                # maximum of 2 threads for tests
+                parallelism-max = 2
+                fixed-pool-size = 1
+            }
+            thread-pool-executor {
+                fixed-pool-size = 1
+            }
+        }
+        default-blocking-io-dispatcher {
+            # minimum of one thread for tests
+            parallelism-min = 1
```

Comment:
I don't think `parallelism-min` and `parallelism` max are valid under `default-blocking-io-dispatcher` , so let's remove them. I think we also should have at least 2 threads in the `fixed-pool-size` as this is for blocking tasks that may take awhile, this will at least let us have two blocking tasks done as the same time.